go.mod

```diff
@@ -42,6 +42,7 @@ require (
 	github.com/google/go-cmp v0.5.4
 	github.com/google/uuid v1.2.0
 	github.com/gosimple/slug v1.9.0
+	github.com/grafana/grafana-aws-sdk v0.0.0-20210112151715-f082803d987a
```

Comment:
Would be nice to tag the github.com/grafana/grafana-aws-sdk, e.g. v0.1.0, to make it easier to understand what version/changes of the lib grafana is using. Probably a good idea to upgrade github.com/grafana/grafana-plugin-sdk-go to the latest version in github.com/grafana/grafana-aws-sdk as well.

Reply:
Good idea - will follow up with that!